taxi/commands/alias.py

```diff
@@ -120,7 +120,7 @@ def list_aliases(ctx, search, backend, used, inactive):
         else:
             project = None
 
-        if not inactive and (not project or not project.is_active()):
+        if not inactive and ((not project or not project.is_active()) or (activity and not activity.is_active())):
```

Comment:
The initial condition was already hard to read but this is making it even harder. ^^ Could you please rename the `inactive` parameter to `include_inactive` and rewrite the condition like so?

```python
project_is_active = project and project.is_active()
activity_is_active = activity and activity.is_active()

if include_inactive or project_is_active and activity_is_active:
    ctx.obj…
```